broadinstitute/pearl
# [JN-546] switching to zendesk help

The ZenDesk site isn't yet public (requires sign in), so I'll wait to merge/release this until it is live for end users.

TO TEST:
1. click the help icon in the admin tool navbar
2. click "help pages"
3. confirm you are taken to zendesk
